PxAPI-2.yml

```diff
@@ -363,6 +382,110 @@ components:
         type: integer
         #Default value depends on site?
   schemas:
+    ConfigResponse:
+      type: object
+      description: API configuration
+      required:
+        - "apiVersion"
+        - "languages"
+        - "defaultLanguage"
+        - "maxDataCells"
+        - "maxCalls"
+        - "timeWindow"
+        - "license"
+      properties:
+        apiVersion:
+          type: string
+          description: The version of then API
+        languages:
+          type: array
+          description: A list of language that exists for the data.
+          items: 
+            $ref: '#/components/schemas/Language'
+        defaultLanguage:

```

Comment:
Is it better to say that the first entry in languages is the default?

Reply:
It think it might be clearer if we explicitly state which language is the default language. Making it easier for consumers to  understand what will be the default lanuage.